tests/ZiraLink.Api.Application.UnitTests/Services/CustomerServiceTest.cs

```diff
@@ -1,11 +1,21 @@
 ﻿
+using Castle.Core.Resource;
```

Comment:
suggest to remove all extra new lines from usings

Reply:
Removed.